package.json

```diff
@@ -4,11 +4,11 @@
   "description": "Feedhenry Raincatcher Angular Repository",
   "main": "index.js",
   "scripts": {
-    "test": "lerna run test --parallel --stream",
+    "test": "lerna run test --parallel --stream --ignore='@raincatcher/datasync-client' --ignore='@raincatcher/wfm' --ignore='@raincatcher/logger'",
```

Comment:
That's not going to scale. We should use different lerna config for tests I think rather than --ignore that needs to specify package names.

Reply:
Agree, the only alternative that I can think of would be having a stricter naming scheme for packages, because there's no way to filter them based on directory

Reply:
We could also just run tests on `@raincatcher/angularjs-*` and `@raincatcher-examples/*` and change any remaining packages to match those

Reply:
Let's merge that and I will pass different lerna config file to this command